src/connector/src/parser/unified/kv_event.rs

```diff
@@ -79,6 +79,9 @@ where
     pub fn access_field(&self, desc: &SourceColumnDesc) -> AccessResult<DatumCow<'_>> {
         match desc.additional_column.column_type {
             Some(AdditionalColumnType::Key(_)) => self.access_key(&[&desc.name], &desc.data_type),
+            // hack here: Get the whole payload as a single column
+            // use a special mark empty slice as path to represent the whole payload
```

Comment:
nit: I think empty path is not "hack" at all, but just reasonable usage? 
```suggestion
            // access empty path to get the whole payload
```

Reply:
I think it is an undefined behavior in prev impl, we always expect `path` a non-empty slice.